KsApi/ServiceType.swift

```diff
@@ -47,6 +47,10 @@ public protocol ServiceType {
   func addImage(file fileURL: URL, toDraft draft: UpdateDraft)
     -> SignalProducer<UpdateDraft.Image, ErrorEnvelope>
 
+  /// Views project after navigating to the via facebook ad deep-link
```

Comment:
Watch comments here, ensure they are correctly written for each function and reflect its use.